localisation/00_zuwc_united_localisation.csv

```diff
@@ -3119,28 +3119,30 @@ EVTOPTA_MNM_8052_white_tiger;To protect home and family, to preserve balance and
 EVTNAME_MNM_8053_jade_serpent;The Temple of the Jade Serpent;;Der Tempel der Jadeschlange;;;;;;;;;;;x
 EVTDESC_MNM_8053_jade_serpent;You have finally arrived at the heart of the Jade Forest. One of the August Celestials, Yu'lon the Jade Serpent, lives here in her temple in the thick of the forest. She is the spirit of wisdom, but also represents spring, youth, growth, and nurturing.\n\nWhen Emperor Shaohao climbed up to Mount Neverest, the Jade Serpent appeared in front of him. Shaohao needed to know, how to keep his land and people safe from the Sundering. She told him to go to the center of Pandaria and find the answer within.;;Ihr seid endlich im Herzen des Jadewaldes angekommen. Eine der himmlischen Erhabenen, Yu'lon, die Jadeschlange, lebt hier in ihrem Tempel mitten im Wald. Sie ist der Geist der Weisheit, steht aber auch f�r Fr�hling, Jugend, Wachstum und Pflege.\n\nAls Kaiser Shaohao auf den Berg Neverest kletterte, erschien die Jadeschlange vor ihm. Shaohao musste herausfinden, wie er sein Land und sein Volk vor der Teilung sch�tzen konnte. Sie sagte ihm, er solle ins Zentrum von Pandaria gehen und die Antwort im Inneren finden.;;;;;;;;;;;x
 EVTOPTA_MNM_8053_jade_serpent;May the Jade Serpent guide me.;;M�ge die Jadeschlange mich leiten.;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
+EVTDESC_WCRAC_1005;Your servants say that an emissary of the Klaxxi has arrived. You welcome him and give the best amber as a gift of goodwill. One thing led to another, and he announces that the Klaxxi believe that you're worth to become a paragon: one of the best of the mantid preserved in amber in case of a great crisis.;;;;;;;;;;;;;x
```

Comment:
```suggestion
EVTDESC_WCRAC_1005;Your servants say that an emissary of the Klaxxi has arrived. You welcome him and offer kypari amber as a gift of goodwill. One thing led to another, and he announces that the Klaxxi believe that you're worth to become a paragon: one of the best warriors of the mantid, preserved forever in amber in case of a great crisis.;;;;;;;;;;;;;x
```

Reply:
> the best warriors

Not only warriors: diplomats, administrators too.

Reply:
Fair point, I had forgotten the paragons were more than just warriors/ assassins. Took that bit out.